tests/bluetooth/audio/mocks/include/bap_unicast_server_expects.h

```diff
@@ -0,0 +1,116 @@
+		     "'%s()' was not called", func_name);                                          \
+	zassert_equal(1, mock_bap_unicast_server_cb_enable_fake.call_count,                        \
+		      "'%s()' was called more than once", func_name);                              \
+												   \
+	IF_NOT_EMPTY(_stream, (                                                                    \
+		     zassert_equal_ptr(_stream, mock_bap_unicast_server_cb_enable_fake.arg0_val,   \
+				       "'%s()' was called with incorrect '%s' value",              \
+				       func_name, "stream");))                                     \
+												   \
+	IF_NOT_EMPTY(_meta, (                                                                      \
+		     /* TODO */                                                                    \
+		     zassert_unreachable("Not implemented");))                                     \
+												   \
+	IF_NOT_EMPTY(_meta_count, (                                                                \
+		     /* TODO */                                                                    \
+		     zassert_unreachable("Not implemented");))                                     \
+} while (0)
+
+#define expect_bt_bap_unicast_server_cb_disable_called_once(_stream)                               \
+do {                                                                                               \
+	const char *func_name = "bt_bap_unicast_server_cb.disable";                                \
+												   \
+	zassert_true(mock_bap_unicast_server_cb_disable_fake.call_count > 0,                       \
+		     "'%s()' was not called", func_name);                                          \
+	zassert_equal(1, mock_bap_unicast_server_cb_disable_fake.call_count,                       \
+		      "'%s()' was called more than once", func_name);                              \
+												   \
+	IF_NOT_EMPTY(_stream, (                                                                    \
+		     zassert_equal_ptr(_stream, mock_bap_unicast_server_cb_disable_fake.arg0_val,  \
+				       "'%s()' was called with incorrect '%s' value",              \
+				       func_name, "stream");))                                     \
+} while (0)
+
+#define expect_bt_bap_unicast_server_cb_release_called_once(_stream)                               \
+do {                                                                                               \
+	const char *func_name = "bt_bap_unicast_server_cb.release";                                \
+												   \
+	zassert_true(mock_bap_unicast_server_cb_release_fake.call_count > 0,                       \
+		     "'%s()' was not called", func_name);                                          \
+	zassert_equal(1, mock_bap_unicast_server_cb_release_fake.call_count,                       \
+		      "'%s()' was called more than once", func_name);                              \
```

Comment:
You have this pattern many places. Could it be replaced by 

```c
	zassert_equal(1, mock_bap_unicast_server_cb_release_fake.call_count,                       \
		      "'%s()' was not called just once: %u", func_name, mock_bap_unicast_server_cb_release_fake.call_count); 
```

If we just want to check against the value `1`, it seems superfluous to first check if it is above 0 before checking if it equal to 1.

Reply:
@MariuszSkamra Are you planning on changing this? If you are, then I'll approve once it has been updated. Since this is just a minor thing, I can also improve as is, if you don't want to change this

Reply:
As you don't insist, let's leave it as it is

Reply:
Ah I understand now the game you play. There you go, fixed this